[
 {
  "author": "dims", 
  "date": "2002-03-06T22:21:08.000000Z", 
  "msg": [
   "Added but commented out a case of SOAP structure within another."
  ], 
  "revision_id": "246619"
 }, 
 {
  "author": "scheu", 
  "date": "2002-03-06T21:41:34.000000Z", 
  "msg": [
   "testArrayM is just like testArrayL, but it does not use multiref"
  ], 
  "revision_id": "246618"
 }, 
 {
  "author": "scheu", 
  "date": "2002-03-06T20:51:12.000000Z", 
  "msg": [
   "Patch for parameter name detection submitted by Thomas Borkel"
  ], 
  "revision_id": "246617"
 }, 
 {
  "author": "dims", 
  "date": "2002-03-06T20:39:40.000000Z", 
  "msg": [
   "Added a test case for Nested complex return values."
  ], 
  "revision_id": "246616"
 }, 
 {
  "author": "butek", 
  "date": "2002-03-06T20:34:17.000000Z", 
  "msg": [
   "I got rid of some of the ignorable exceptions from the functional-tests.  I only went", 
   "after the biggest offender - the multithread test - but this sort of fix should be done", 
   "everywhere.  Here's the philosophy I followed:", 
   "- if a catch block is rethrowing the exception, don't log it (it will probably be logged", 
   "elsewhere)", 
   "- if a catch block throws a different exception, log the original, but log it as debug, not", 
   "error; this bit of code is in error, but somebody higher up the call chain may handle it,", 
   "in which case it is NOT an error.", 
   "This is a change in philosophy that we might want to discuss."
  ], 
  "revision_id": "246615"
 }
]